### PR TITLE
Individual activity 4

### PR DESCRIPTION
**Pull request by ijromerob**

### CODE REVIEW

File: src/test/product.test.js
Comment (jcampos-94) [inline src/test/product.test.js:1]: This needs to be erased.

File: src/js/ExternalServices.mjs
Comment (jcampos-94) [inline src/js/ExternalServices.mjs:1]: It looks good, but a function here or in utils.mjs to delete previous alerts automatically would be helpful.

**Review by jcampos-94 — CHANGES_REQUESTED** [review]
Added a few comments. Also, the success page doesn't load. I think you forgot to edit vite.config.js

**Comment by ijromerob** [comment]
this branch was not used

**Pull request closed without merging**